invenio_jobs/resources/config.py

```diff
@@ -37,15 +38,40 @@ class JobsResourceConfig(ResourceConfig, ConfiguratorMixin):
     url_prefix = "/jobs"
     routes = {
         "list": "",
-        "item": "/<id>",
+        "item": "/<job_id>",
     }
 
     # Request parsing
     request_read_args = {}
-    request_view_args = {"id": ma.fields.Int()}
+    request_view_args = {"job_id": ma.fields.Int()}
     request_search_args = JobsSearchRequestArgsSchema
 
     error_handlers = {
         **ErrorHandlersMixin.error_handlers,
         # TODO: Add custom error handlers here
     }
+
+
+class RunsResourceConfig(ResourceConfig, ConfiguratorMixin):
+    """Runs resource config."""
+
+    # Blueprint configuration
+    blueprint_name = "job_runs"
+    url_prefix = ""
+
+    routes = {
+        "runs": "/jobs/<job_id>/runs",
+        "run_item": "/jobs/<job_id>/runs/<run_id>",
+        "logs": "/jobs/<job_id>/runs/<run_id>/logs",
+    }
```

Comment:
```suggestion
        "actions_stop": "/jobs/<job_id>/runs/<run_id>/actions/stop",
    }
```

minor: to have an explicit endpoint for stopping/cancelling a task

Reply:
Should we consider adding a `pause` endpoint for pausing a task at this point @slint ?

Reply:
@Samk13 I've also logged some thoughts on how/where we could be tracking the task status [in the RFC](https://codimd.web.cern.ch/qjdmyWRvRiqujzp8BC6PgA?both#Detailed-job-status-tracking).

Reply:
@slint  Good, if the status comes with the run payload, then it's enough for us.
Regarding the link, I can't log in with KTH institution credentials, so I can't see what's in the RFC.

Reply:
@karkraeg @mkloeppe, how did you manage to login through the CERN SSO? Did you use a Google account, or eduGAIN?

Reply:
@slint eduGAIN works

Reply:
@slint It is possible that KTH is not integrated with EduGain, which might explain the login issue.